### ISSUE TITLE
Global setUp not TestCase based, and DatabaseTransactions to rollback data created during testCase but keep schema (Laravel)

### ISSUE THREAD

**Issue by romanstingler** [sebastianbergmann/phpunit#5623]
In Laravel using PHPUnit, I aim to execute migrations only once during the initial setup before any other test runs. 
In my case I have the migrations from the project and external migrations like

```$this->artisan('migrate', ['--path' => 'vendor/common_configuration/database/migrations/']);```

Due to the fact that my schema doesn't change during testing I want to run the migrations (external + internal) one and keep the DB (postgres) persistent. 

On top of that, I want to use the `DatabaseTransactions` trait to start a transaction on the start of a testCase and rollback at the end, 
so that seeds and other created Data is deleted at the end, but my schema is kept in place.

Any advice?

**Comment by romanstingler** [comment]
Thank you very much, tomorrow I will take a look at it.

**Comment by sebastianbergmann** [comment]
Laravel-specific topics are out of scope for this issue tracker.
